src/taco-utils/resources/en/resources.json

```diff
@@ -76,5 +76,22 @@
     "_TelemetryCurrentlyOptedInPrompt.comment": "This is the string presented to the user before asking to opt in for telemetry when he has currently opted in.",
     "TelemetryCurrentlyOptedOutPrompt": "Would you like to allow us to collect your usage data? [y/n]",
-    "_TelemetryCurrentlyOptedOutPrompt.comment": "This is the string presented to the user before asking to opt in for telemetry when he has currently opted out."
+    "_TelemetryCurrentlyOptedOutPrompt.comment": "This is the string presented to the user before asking to opt in for telemetry when he has currently opted out.",
+    "CordovaCmdNotFound": "Cordova not found, please run 'npm install –g cordova' to install Cordova globally",
+    "_CordovaCmdNotFound.comment": "Error message shown when no Cordova is found on the machine",
+    "CordovaCommandFailed": "'cordova {1}' failed with exit code {0}",
+    "_CordovaCommandFailed.comment": "Error message passed back after invoking Cordova fails. {0} is the exit code, and {1} is the arguments passed to it",
+    "CordovaCommandFailedWithError": "'cordova {0}' failed.",
+    "_CordovaCommandFailedWithError.comment": "Error message passed back after invoking Cordova fails. {0} is the exit code, and {1} is the arguments passed to it",
+    "CordovaCommandUnhandledException": "Cordova threw an unhandled exception, check http://go.microsoft.com/fwlink/?LinkId=623759 for possible details.",
+    "_CordovaCommandUnhandledException.comment": "Error message reported when Cordova throws an unhandled exception",
+    "CommandCreateTacoJsonFileCreationError": "Error: Creation of taco.json file failed",
+    "_CommandCreateTacoJsonFileCreationError.comment": "This message is displayed when the taco.json file creation fails.",
+    "CommandCreateTacoJsonFileWriteError": "Failed to write to {0}",
+    "CommandCreateOnlyLocalCustomWww": "Error: Only local paths for custom www assets are supported.",
+    "_CommandCreateOnlyLocalCustomWww.comment": "Error reported when the specified path to the --copy-from or --link-to parameters is a web URL.",
+    "InvalidCordovaWithNode5": "Incompatible configuration: Unable to use Cordova < 5.4.0 with Node >= 5.0.0. Either upgrade to a newer version of Cordova, or revert to an older version of node. See http://go.microsoft.com/fwlink/?LinkID=618471 for more details.",
+    "_InvalidCordovaWithNode5.comment": "Error message printed when trying to use Cordova < 5.4.0 with Node >= 5.0.0.",
+    "CommandInstallCordovaTooOld": "Error: The current project uses Cordova version {0}. 'taco install-reqs' supports projects using Cordova version {1} or higher.<br/><br/>If your project was created using a Kit, consider moving to a kit that has a more recent version of Cordova. If your project was created using Cordova directly, consider updating the installation of Cordova. For more information, please refer to http://go.microsoft.com/fwlink/?LinkId=618544.",
```

Comment:
This error feels too TACO CLI specific to be a taco-utils candidate. Maybe get the relevant command as a param?